### PR TITLE
fix: add ValidateHttpRequestSync method for synschronous request validation

### PR DESCRIPTION
**Pull request by commoddity**
Implements the feature requested in: https://github.com/pb33f/libopenapi-validator/issues/79

### CODE REVIEW

**Comment by daveshanley** [comment]
Missing a few lines of coverage, as soon as it's bumped, I can merge this in!

**Comment by commoddity** [comment]
> Missing a few lines of coverage, as soon as it's bumped, I can merge this in!

Thank you. Have just pushed a commit that I believe should cover those 3 missing lines.

File: requests/validate_body.go [hunk 1]
```diff
@@ -31,6 +32,7 @@ func (v *requestBodyValidator) ValidateRequestBody(request *http.Request) (bool,
 
 	operation := helpers.ExtractOperation(request, pathItem)
 	if operation == nil {
+		fmt.Println("HERE!!!!")
```
Comment (daveshanley) [inline requests/validate_body.go:35]: This needs to be removed!

**Review by daveshanley — CHANGES_REQUESTED** [review]
Looks good to me, if you remove that print statement on line 25 in `validate_body.go` then it's good to go!

File: requests/validate_body.go (hunk 1 above)
Comment (commoddity) [reply requests/validate_body.go]: Oops! It certainly does. Removed.

**Comment by commoddity** [comment]
> Looks good to me, if you remove that print statement on line 25 in `validate_body.go` then it's good to go!

Sounds good, I've updated to remove that print.

**Review by daveshanley — APPROVED** [review]
LGTM!

**Pull request merged**